driver-core/src/main/java/com/datastax/driver/core/Cluster.java

```diff
@@ -744,6 +745,22 @@ public Builder withPort(int port) {
         }
 
         /**
+         * Allows client to connect to server using latest development protocol version,
+         * which is currently in beta.
+         * <p/>
+         * Use with caution, refer to the server and protocol documentation for the details
+         * on latest protocol version.
+         *
+         * @param allowBetaProtocolVersions {@code true} if connecting with beta protocol version should be allowed,
+         *                                  {@code false} otherwise
+         * @return this Builder.
+         */
+        public Builder setAllowBetaProtocolVersions(boolean allowBetaProtocolVersions) {
```

Comment:
Following the example of other boolean properties in this builder, I would suggest turning this method into a more simple `allowBetaProtocolVersions()` that would internally set the corresponding flag to `true` (with no possibility of turning back to `false`).